python/ray/workflow/tests/test_dag_to_workflow.py

```diff
@@ -108,6 +108,28 @@ def f(a):
     assert ray.get(*result3) == 10
 
 
+def test_dereference_object_refs(workflow_start_regular_shared):
+    """Ensure that object refs are dereferenced like in ray tasks."""
+
+    @ray.remote
+    def f(obj_list):
+        assert isinstance(obj_list[0], ray.ObjectRef)
+        assert ray.get(obj_list[0]) == 42
+
+    @ray.remote
+    def g(x, y):
+        assert x == 314
+        assert ray.get(y) == [2022]
```

Comment:
check y is a list of object ref. Otherwise, y could be an object ref and ray.get(y) is a list.